docs/src/main/asciidoc/writing-extensions.adoc

```diff
@@ -2334,7 +2336,7 @@ A feature can be registered in a <<build-step-processors>> method that produces
     }
 ----
 
-The name of the feature should only contain lowercase characters, words are separated by dash; e.g. `security-jpa`.
+The name of the feature should only contain lowercase characters, words are separated by dash; e.g. `quarkus-security-jpa`.
```

Comment:
not sure why `quarkus-` added here. That is not related to the requirement.

Reply:
I'm sorry—that's an artifact of some other work I was doing. I'm reverting this change now.